benchmarks/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "benchmarks",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Benchmark for @joshdb providers",
+  "author": "@joshdb",
+  "dependencies": {
+    "@joshdb/provider": "2.0.0-next.6cc7d77.0"
+  }
+}
```

Comment:
Could probably add scripts here for the benchmarks to run.

Reply:
It was a bit, weird to do. I tried but it wouldn't output dist correctly

Reply:
I see, we can look into that later then.